www/codePush.ts

```diff
@@ -63,6 +63,14 @@ class CodePush implements CodePushCordovaPlugin {
      * !!! This function is called from the native side, please make changes accordingly. !!!
      */
     public reportStatus(status: number, label: string, appVersion: string, deploymentKey: string, previousLabelOrAppVersion?: string, previousDeploymentKey?: string) {
+       if ((!label || label === previousLabelOrAppVersion) &&
+           appVersion === previousLabelOrAppVersion &&
+           deploymentKey === previousDeploymentKey) {
```

Comment:
same comment as above...

Shouldn't you check either `label` or `appVersion` against `previousLabelOrAppVersion` and not both? If `label` is defined and is equal to `previousLabelOrAppVersion` and `appVersion` which I am assuming is not same as `label` since it can't be, in that case it is not going to be equal to `previousLabelOrAppVersion` then it should return, shouldn't it?